backend/src/mailer/text/transactionReceived.ts

```diff
@@ -25,6 +26,9 @@ ${data.memo}
 Bitte antworte nicht auf diese E-Mail!
 
 Mit freundlichen Grüßen,
-dein Gradido-Team`,
+dein Gradido-Team
+
+
+Link zur Übersicht: <a href="${data.overviewURL}">Deine Übersicht</a>`,
```

Comment:
Is the design approbriate?

Reply:
Actually no, since its a text email - no html tags here

Reply:
How I can get a link in the text then?

Reply:
I removed the HTML.